.github/workflows/release.yml

```diff
@@ -16,6 +16,11 @@ jobs:
     permissions:
       id-token: write
     steps:
+      - name: Validate tag
+        run: |
+          echo "Validating that the tag is in the correct format"
+          echo "${{ github.event.release.tag_name }}" | grep -P '^v(\d+)\.(\d+)\.(\d+)(-(preview|rc)\d+$|$)'
```

Comment:
Could we have different checks, so that preview-releases always have to include `preview` or `rc`, but normal releases can never have them? :thinking:

Reply:
ooo did not think of that, can have a look, I assume theres a boolean that's part of the `release` object

Reply:
added a conditional now, and did some testing using act. Don't know if it's worth to keep the files around for testing the pipeline